src/components/MainHeader/MainHeader.module.scss

```diff
@@ -90,6 +91,7 @@
 @include respond-to(medium-screen) {
   .topBar {
     height: $responsive-header-topbar-height;
+    max-width: 100%;
```

Comment:
This is unnecessary as 94rem is the max width for all sizes

Reply:
Cool